pkg/trimaran/peaks/peaks_test.go

```diff
@@ -0,0 +1,556 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package peaks
+
+import (
+	"context"
+	"encoding/json"
+	"math"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+
+	"github.com/paypal/load-watcher/pkg/watcher"
+	"github.com/stretchr/testify/assert"
+	testutil "sigs.k8s.io/scheduler-plugins/test/util"
```

Comment:
can you sort the imports into several sections and separate by empty lines?

- core Golang imports like `math`
- upstream imports like loadwatcher, k8s.io
- imports from this repo (sigs.k8s.io/scheduler-plugins)

Reply:
Rearranged the imports